tests/std/tests/VSO_0000000_instantiate_containers/test.cpp

```diff
@@ -46,6 +46,20 @@ using namespace std;
 
 int main() {} // COMPILE-ONLY
 
+// FUNCTION TEMPLATE For_each_tuple_element

```

Comment:
This contains a typo (`for_each_tuple_element` here is lowercase), but actually this line should simply be removed as our (non-tr1) test code doesn't use ALL-CAPS BANNERS, and avoiding them in the tests makes it slightly easier to globally search for product definitions:

```suggestion
```